### ISSUE TITLE
Empty contrib-group in conversionStage 5

### ISSUE THREAD

**Issue by fabiobatalha** [pkp/ots#117]
We are receiving a empty contrib-group in the conversionStage 5.

It is invalid against JATS and JATS4M

The job related to this issue is: 2651

It is enough to give the job number as reference?

In this same document the authors metadata were tagged as affiliations.

**Comment by axfelix** [comment]
OK, I can duplicate this, it's a very strange edge case. Basically, Cermine failed to find the authors and only populated `contrib-group` with `aff` tags, and we always move the `aff` tags out of `contrib-group` to make Texture happy. I've never seen another document that only finds `aff` but no actual `authors`, so I haven't implemented a check to make sure that `contrib-group` isn't empty after moving the `aff` tags out.

I guess I need to do that...

**Comment by axfelix** [comment]
@kaschioudi, I'm struggling to tell why this fix I just added isn't working, if you have the chance to take a look: https://github.com/pkp/ots/blob/master/module/MergeXMLOutputs/src/MergeXMLOutputs/Model/Converter/Merge.php#L231

**Comment by kaschioudi** [comment]
@axfelix do you still need me to look into this? Please me know.

**Comment by axfelix** [comment]
Yup, if you don't mind -- I was stumped and have put it down for now.
